pages/grants.js

```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
-                new Urbit developers provided by members of the Urbit community.
-                They're one of the best ways to level up your skills, and often
-                lead to full-time jobs.
-              </p>
-              <Link href="/grants/apprenticeships" passHref>
-                <a className="button-sm bg-green-400 text-white mb-8 max-w-fit">
-                  Become an Apprentice
-                </a>
-              </Link>
+              <div className="flex flex-column flex-wrap md:flex-nowrap grid-cols-2 gap-x-12  pb-12">
 
-              {/* Bounties */}
-              <p className="mb-4">
-                <b>Bounties</b> are contracts for work provided by either the
-                Urbit Foundation or from trusted partners in our ecosystem.
-              </p>
-              <div className="flex flex-wrap">
-                <Link href="/grants/bounties#post-a-bounty" passHref>
-                  <a className="button-sm bg-yellow-300 text-black mr-2 max-w-fit">
-                    Post a Bounty
-                  </a>
-                </Link>
-                <Link href="/grants/bounties" passHref>
-                  <a className="button-sm bg-wall-400 text-white max-w-fit">
-                    Learn more
-                  </a>
-                </Link>
-              </div>
+                <div className="md:basis-1/2">
+                  {/* Proposals */}
+                    <h3>Proposals</h3>
+                    <p className="mb-4">
+                    Get funding for your project.  We fund all kinds projects, not strictly technical ones,
+                    so don't hesitate to pitch your idea!
+                    </p>
+                  <Link href="/grants/proposals" passHref>
+                    <a className="button-sm bg-blue-400 text-white mb-8 max-w-fit">
+                      Submit a Proposal
+                    </a>
+                  </Link>
+
+                  {/* Apprenticeships */}
+                    <h3>Apprenticeships</h3>
+                    <p className="mb-4">
+                     Practical learning opportunities for
+                    new Urbit developers provided by members of the Urbit community.
+                    They're one of the best ways to level up your skills, and often
+                    lead to full-time jobs.
+                  </p>
+                  <Link href="/grants/apprenticeships" passHref>
+                    <a className="button-sm bg-green-400 text-white mb-8 max-w-fit">
+                      Become an Apprentice
+                    </a>
+                  </Link>
+                </div>
+
+                <div className="md:basis-1/2">
+                  {/* Bounties */}
+                  <h3>Bounties</h3>
+                  <p className="mb-4">
+                  Contracts for work provided by either the
+                    Urbit Foundation or from trusted partners in our ecosystem.
```

Comment:
```suggestion
                    Urbit Foundation or from trusted partners in the Urbit universe.
```
I thought we weren't using the word "ecosystem"

Reply:
This is one of those things like Mars that's contextual.  People know what a software ecosystem is, a software universe is an unfamiliar term.  So I'd like to say "ecosystem" on public facing pages and on internal communications we can use all our in-house terms.

<img width="803" alt="Screen Shot 2022-07-18 at 11 44 18 AM" src="https://user-images.githubusercontent.com/364577/179581488-ac9be769-1560-45e4-ae6d-757840928178.png">